python/ccf/clients.py

```diff
@@ -179,28 +183,16 @@ def get_curve(ca_file):
 
 class CurlClient:
     """
-    We keep this around in a limited fashion still, because
-    the resulting logs nicely illustrate manual usage in a way using the requests API doesn't
+    Curl client.
```

Comment:
Having a short description repeating the name of the class isn't necessary.

Reply:
Addressed in #1429